modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -221,7 +229,8 @@ public CompletableFuture<Void> startAsync() {
 
                         handleClusterState((ClusterStateMessage) message, sender, correlationId);
                     } else if (message instanceof CancelInitMessage) {
-                        handleCancelInit((CancelInitMessage) message);
+                        // I think the none of the cancel iniØt messages will have correlation id but...
```

Comment:
I think we don't need to report any errors back here. You are right, none of these message have a correlation ID.

Reply:
Do you think its worth to keep the error handling code just to be complete or should I remove it??

Reply:
I would prefer to remove it, it's unnecessary